install-dependencies.sh

```diff
@@ -90,6 +90,7 @@ fedora_packages=(
     glibc-langpack-en
     lld
     xxhash-devel
+    rpm-build
```

Comment:
Not strictly necessary since it's in tools/toolchain/Dockerfile (but okay to have it here).